docs/getting-started/ios/setup.md

```diff
@@ -56,9 +43,20 @@ The most interesting file is `fastlane/Fastfile`, which contains all the informa
 
 _fastlane_ created all the required files for you, now you can go ahead and customise it to generate screenshots or to automatically distribute new builds
 
-- [_fastlane_ screenshots for iOS and tvOS](screenshots.md)
-- [iOS Beta deployment using _fastlane_](beta-deployment.md)
-- [iOS App Store deployment using _fastlane_](appstore-deployment.md)
+- [Generate localized iOS screenshots for the App Store](screenshots.md)
```

Comment:
no more tvOS?

Reply:
We don't officially support tvOS currently 😢